text_mining.py

```diff
@@ -0,0 +1,82 @@
+def tweet_list(hashtag, number):
+    """
+    Takes in a string and value find and returns a list of tweets the value
+    long that contain the string
+    """
+    from pattern.web    import Twitter
+
+    t = Twitter()
+    i = None
+    tweets = []
+    for j in range(number):
+        for tweet in t.search(hashtag, start=i, count=1):
+            # print tweet.text
+            # print
+            tweets.append(tweet.text)
+            i = tweet.id
+    return tweets
+
+def hastag_setiment(hashtag):
+    """
+    Takes in a hashtag and returns the average sentiment of a number of tweets
+    with that hashtag
+    """
+    from pattern.en import *
```

Comment:
See above; importing functions at the top of the python file is preferred.
Also--it's fine here I think, but in general importing * is a bad idea. Lots of libraries will have many different functions, and weird things can happen if multiple functions get the same names.